docs/decisions/0022-create-waffle-switch.rst

```diff
@@ -0,0 +1,58 @@
+###############################################################
+Integration of Waffle Switch for XQueue Submission
```

Comment:
I think this ADR should be in the https://github.com/openedx/edx-platform/tree/58834c6bdc20fae182eca4aa4ce7402cc14dfd47/xmodule/docs/decisions directory, and the file name should include "xqueue" in it (we have a ton of waffle switches, so the title "create-waffle-switch" is really not specific enough)